app/packs/review/_components/UnreviewedPack.tsx

```diff
@@ -8,41 +18,199 @@ import {
 	CardHeader,
 	CardTitle
 } from '@/components/ui/card'
-import { UnreviewedPackDetails } from './UnreviewedPackDetails'
+import { QuestionPackDetails } from '../../_components/QuestionPackDetails'
+import { Textarea } from '@/components/ui/textarea'
+import { toast } from '@/components/ui/use-toast'
+import { Check, X, Loader2 } from 'lucide-react'
+import { Button } from '@/components/ui/button'
+import { Badge } from '@/components/ui/badge'
+import { packMap } from '@/types'
+import { useState } from 'react'
+
+import { PackType } from '@/types'
 
-export interface QuestionPackProps {
-	id: number
-	featured?: boolean // popular
+interface UnreviewedPackProps {
+	id: string
 	name: string
 	description: string
-	likes: number
 	type: string
```

Comment:
We can do the same as before & set this type to PackType from @prisma/client (remove the one from @/types), that way we can get rid of the extra casting on line 115
```suggestion
	type: PackType
```